pkg/epinio/index.ts

```diff
@@ -3,9 +3,21 @@ import { IPlugin, OnNavToPackage, OnNavAwayFromPackage } from '@shell/core/types
 import epinioStore from './store/epinio-store';
 import epinioMgmtStore from './store/epinio-mgmt-store';
 import epinioRoutes from './routing/epinio-routing';
+import { MANAGEMENT } from '@shell/config/types';
 
 const onEnter: OnNavToPackage = async(store, config) => {
   await store.dispatch(`${ epinioMgmtStore.config.namespace }/loadManagement`);
+
+  const serverVersionSettings = store.getters['management/byId'](MANAGEMENT.SETTING, 'server-version');
+  const res = await store.dispatch(`epinio/request`, { opt: { url: `/api/v1/info` } });
+
+  await store.dispatch('management/load', {
```

Comment:
This is fine in the standalone case, but will break rancher manager when the plugin is embedded. We're a fair bit away from getting that to work though, so we just need to track that we need to fix this in a new github issue

Reply:
@scures Was there an issue in `epinio/ui` to track this?